src/tls.c

```diff
@@ -2305,6 +2305,313 @@ int TLSX_SNI_GetFromBuffer(const byte* clientHello, word32 helloSz,
 
 #endif /* HAVE_SNI */
 
+/******************************************************************************/
+/* Trusted CA Key Indication                                                  */
+/******************************************************************************/
+
+#ifdef HAVE_TRUSTED_CA
+
+/** Creates a new TCA object. */
+static TCA* TLSX_TCA_New(byte type, const byte* id, word16 idSz, void* heap)
```

Comment:
Do you think `TCA` might have naming conflicts with customer apps? Perhaps something like `wolfTCA` would be better?

Reply:
`struct TCA` and the TCA functions are all internal APIs. If we change this we should change SNI and ALPN.